### ISSUE TITLE
Move netlify to a ssc owned account ?

### ISSUE THREAD

**Issue by Carreau** [jupyter/jupyter.github.io#768]
Currently the deployed preview netlify is owned by me, free account, limited to 3 collaborators. 

I think it woudl be good to move it to a team managed by the SSC

**Comment by jasongrout** [comment]
+1 to moving to an institutional account. @choldgraf pointed out in the linked issues above that Netlify has an [Open-source program](https://www.netlify.com/open-source/) we could apply for. The main change is that we need to advertise "This site is powered by Netlify" with a link to netlify (or use one of [their badges](https://www.netlify.com/about/#badges).

As another option, since we have our domains at CloudFlare, we could also set up serving/previews through Cloudflare workers.

FYI, I think jupyter.org is actually *hosted* using github pages, even though the previews use netlify. If GitHub built a [preview feature for GitHub pages](https://github.com/orgs/community/discussions/7730), we could just use that.

**Comment by jasongrout** [comment]
If we just use netlify for previews, but not for hosting, I'm not sure if we need to advertise the site is powered by Netlify? Or maybe just the previews need to advertise it?

**Comment by choldgraf** [comment]
I suggest putting a thank you on an acknowledgement page or in a small text bit in the footer or something. Or we can just pay $20 a month for netlify it's really not that much haha

**Comment by Carreau** [comment]
I cannot add anyone to the netlify team, and I don't seem to be able to transfer the project; (unless i migrate to paid plan), so I think we need to setup a new team, and I can then erase my deployment by coordinating with someone controlling the other

**Comment by Carreau** [comment]
I'm also assuming you want netlify to avoid using an action like https://github.com/rossjrw/pr-preview-action that deploy on the same domain ?

**Comment by choldgraf** [comment]
I've seen other projects use that action and seem to like it - I think @brian-rose uses this for pythia, maybe he has thoughts? Either way +1 from me for trying that

**Comment by manics** [comment]
It sounds like https://github.com/rossjrw/pr-preview-action deploys to the same domain, but under a prefix? That sounds risky, if someone opens a malicious PR (maybe after opening a real PR to get past the "new contributor" check) they could push malware and a direct link to it would appear legitimate.

**Comment by choldgraf** [comment]
Yes, that's a consideration to take as well. It would have a URL like this:

```
https://[owner].github.io/[repo]/pr-preview/pr-[number]/
```

You're worried that `pr-preview` and `pr-[number`] would not be clear enough to people, and that people would think it was a legitimate jupyter page?

**Comment by jasongrout** [comment]
@Carreau, I set up a projectjupyter@gmail.com netlify account (credentials in 1password) and hooked it up to this repo, so now it looks like we have two netlify previews going on. I think it would be okay for you to delete your netlify project now.

**Comment by jasongrout** [comment]
For the record: to set up a new project on netlify:

Go to https://github.com/organizations/jupyter/settings/installations, click "configure" next to Netlify, scroll down to Repository access, then select the repo to grant access, then Save

In Netlify, go to https://app.netlify.com/teams/projectjupyter/projects, click "Add new project", then "GitHub". Change the org to Jupyter, then click the repo you just granted access to. It may have you log in via github.

Give the project a name. Add Build Command and Publish Directory as specified in the readme build instructions.

Here is then how I configure the project:

- Visibility: https://app.netlify.com/projects/preview-jupyter-website/configuration/general#visitor-access - set to Public
- Notifications: https://app.netlify.com/projects/preview-jupyter-website/configuration/notifications#deploy-notifications - delete email and "deploy summary" items
- "Lock to stop auto-publishing" on the first build of main branch - eh, might as well since we are only using previews?
- Access and Security > Firewall Traffic Rules > Published Deploys: Block All Traffic (since we are only using the pr previews)

Also:
- AI enablement - go to team settings and disable AI - https://app.netlify.com/teams/projectjupyter/settings/general#ai-enablement

**Comment by Carreau** [comment]
I"ve disabled my version for now, if there is no issues I can delete in a few weeks.

**Comment by jasongrout** [comment]
Thanks so much for setting up this preview years ago! I'll close this issue as resolved.

**Comment by brian-rose** [comment]
> I've seen other projects use that action and seem to like it - I think [@brian-rose](https://github.com/brian-rose) uses this for pythia, maybe he has thoughts? Either way +1 from me for trying that

Pythia doesn't use https://github.com/rossjrw/pr-preview-action but has its own reusable workflows that accomplish much the same thing, with additional workarounds to support previews from forks, even though [this is a bad idea](https://github.com/rossjrw/pr-preview-action/pull/6).
